src/utils/winDimensions.js

```diff
@@ -0,0 +1,63 @@
+import {canAccessWindowTop, internal as utilsInternals} from '../utils.js';
+
+const CHECK_INTERVAL_MS = 20;
```

Comment:
should we make this configurable? Does this seem aggressive?

Reply:
In practice it's refreshed on every auction, as that's when modules query for dimensions.